.github/workflows/tools.yml

```diff
@@ -81,11 +79,9 @@ jobs:
             subsystem: test
             label: test
             run: |
-              NEW_VERSION=$(npm view postject dist-tags.latest)
-              CURRENT_VERSION=$(node -p "require('./test/fixtures/postject-copy/node_modules/postject/package.json').version")
-              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
+              NEW_VERSION=$(./tools/dep_updaters/update-eslint.sh | tail -n1)
+              if [ $? = 0 ]; then
                 echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
-                ./tools/dep_updaters/update-postject.sh
               fi
```

Comment:
I finally use the approach where I print the version at the end of the script.

Let me know if it makes sense @aduh95 @RaisinTen ^^